sdk/lib/pseh/i386/pseh3.c

```diff
@@ -156,20 +164,21 @@ static inline
 LONG
 _SEH3$_InvokeFilter(
     volatile SEH3$_REGISTRATION_FRAME *RegistrationFrame,
-    PVOID Filter)
+    PVOID Filter,
+    int HandlerType)
 {
     LONG FilterResult;
 
-    if (RegistrationFrame->ScopeTable->HandlerType == _SEH3$_NESTED_HANDLER)
+    if (HandlerType == _SEH3$_NESTED_HANDLER)
```

Comment:
Within this `_SEH3$_InvokeFilter` function, is it now guaranteed that the provided `HandlerType` will still be equal to `RegistrationFrame->ScopeTable->HandlerType`, or can they now be different also?